lib/twine/cli.rb

```diff
@@ -66,6 +66,10 @@ module CLI
         DESC
         default: :all
       },
+      :section => {
+        switch: ['-i', '--section SECTION'],
+        description: 'Appy the specified section to use when consuming a localization file.'
```

Comment:
Do we need some help text that this is only useful for when `--consume-all` is set and if this string isn't already in another section? As in: This section will only be applied to new strings that aren't already found in the twine file.